README.md

```diff
@@ -22,28 +32,147 @@ The Vault library is a required dependency of the [EOSIO SDK for Swift: Vault Si
 - [Want to Help?](#want-to-help)
 - [License & Legal](#license)
 
+## About Signature Providers
+
+The Signature Provider abstraction is arguably the most useful of all of the [EOSIO SDK for Swift](https://github.com/EOSIO/eosio-swift) providers. It is responsible for:
+
+* finding out what keys are available for signing (`getAvailableKeys`), and
+* requesting and obtaining transaction signatures with a subset of the available keys (`signTransaction`).
+
+By simply switching out the signature provider on a transaction, signature requests can be routed any number of ways. Need a signature from keys in the platform's Keychain or Secure Enclave? [Configure the `EosioTransaction`](https://github.com/EOSIO/eosio-swift#basic-usage) with this signature provider. Need software signing? Take a look at the [Softkey Signature Provider](https://github.com/EOSIO/eosio-swift/blob/master/Sources/EosioSwiftSoftkeySignatureProvider/EosioSoftkeySignatureProvider.swift) component of [EOSIO SDK for Swift](https://github.com/EOSIO/eosio-swift).
+
+All signature providers must conform to the [EosioSignatureProviderProtocol](https://github.com/EOSIO/eosio-swift/blob/master/Sources/EosioSwift/EosioSignatureProviderProtocol/EosioSignatureProviderProtocol.swift) Protocol.
+
 ## Prerequisites
 
-* Xcode 10 or higher
-* CocoaPods 1.5.3 or higher
-* For iOS, iOS 11.3+
+* Xcode 11 or higher
+* CocoaPods 1.9.3 or higher
+* Swift Package Manager swift-tools 5.2 or higher
```

Comment:
Is "swift-tools..." supposed to be a new bullet point, or same as is here?

Reply:
Same as here.  swift-tools is the main versioned component of Swift Package Manager.  Maybe parentheses or leave of the swift-tools portion and just say 5.2 or higher?

Reply:
I think I will just drop "swift-tools" out of this line and say "Swift Package Manager 5.2 or higher"